tutorials/quantum_simulation/Spin chain of 3 atoms in XY mode.ipynb

```diff
@@ -34,7 +34,7 @@
   },
```

Comment:
<p>There is a <code>H_{XY} = \frac{1}{2}\sum_{i</code>  right in the middle of the text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>